packages/helper-settings/components/stats.js

```diff
@@ -0,0 +1,14 @@
+import { h } from 'preact';
+
+const options = {
+  year: 'numeric',
+  month: 'long',
+  day: 'numeric',
+};
```

Comment:
Is any of this being used?

Reply:
Sorry, I should have paid more addention to the diff before asking for review. To answer your quesiton, no it's not. I was thinking of showing the date of since clicks are tracked, but to keep the wording simple I removed it.

Reply:
What's `h` used for?

Reply:
This is needed to make JSX work. JSX is not part of the JavaScript spec so we have to manual import it. I think there are also ways to automate this using a webpack plugin, but since the usage of preact is quite low it might not worth the effort.